packages/typespec-azure-resource-manager/lib/common-types/managed-identity.tsp

```diff
@@ -1,22 +1,10 @@
 using Azure.ResourceManager.Private;
 
-namespace Azure.ResourceManager.Foundations;
+namespace Azure.ResourceManager.CommonTypes;
```

Comment:
for 'ManagedServiceIdentityType`, how are we handling the 'space' issue in `systemAssignedIdentity, userAssignedIdentity` - where i initially had a space and is now required to have no space?